cache/mise.toml

```diff
@@ -0,0 +1,2 @@
+[tools]
+ruby = "latest"
```

Comment:
Should we also add Elixir & Erlang? I believe the server has them scoped to the `server/` directory.